src/clientUtils.js

```diff
@@ -178,7 +176,7 @@ client.BaseClient = class {
 	 * Open the connection and start responding to event listeners.
    * Non-interactive, runnable in both tests and called by startChat.
 	 */
-  start() {
+  init() {
```

Comment:
I can see `init` being more clear about what the function is doing, and less confusing with the existing `startChat` message. We don't have a corresponding `stop` also.

Reply:
Yes, also it was clashing with the cliClient as it already had a start() function

Reply:
whoa nice catch. i don't doubt it, i remember too many `startBlah` functions and thought we needed better names :)